dashboards/authentication/di-auth-ticf/di-auth-ticf.json.tpl

```diff
@@ -0,0 +1,581 @@
+{
+  "metadata": {
```

Comment:
Sorry, but I'm not sure whether it will complain that the metadata section is empty, might just be worth replacing with...
`"metadata": {
    "configurationVersions": [
      7
    ],
    "clusterVersion": "1.292.40.20240522-160528"
  },`

Reply:
Yeah I saw some with and some without, so I wasn't sure if it was necessary, but I'll chuck it in :-)

Reply:
We discussed and we're going to leave this out, it doesn't appear to be necessary